src/components/FeatureFlagModal/FeatureFlagModal.tsx

```diff
@@ -218,16 +221,47 @@ function FeatureFlagOption({ value, variant, featureFlag, label }: FeatureFlagPr
   )
 }
 
+interface DynamicConfigDropdownProps {
+  configName: DynamicConfigName
+  label: string
+  options: any[]
+  selected: any[]
+}
+
+function DynamicConfigDropdown({ configName, label, options, selected }: DynamicConfigDropdownProps) {
+  const updateConfig = useUpdateConfig()
+  const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const selectedValues = Array.from(e.target.selectedOptions, (opt) => Number.parseInt(opt.value))
```

Comment:
so the values are not always going to be integers 🤔 so the default convention is that `select` values are generally strings and its up to the consumer to parse the string into the selected format correctly, maybe allow there to be a `prop` like `parser` that defaults to `Number.parseInt`?